modules/installation/pages/proxy-setup.adoc

```diff
@@ -194,46 +194,72 @@ filename: "pxelinux.0"
 [[replace-susemgrproxy]]
 == Replace a {productname} Proxy
 
-A proxy does not contain any information about the clients that are connected to it.
-Therefore, a proxy can be replaced by a new one at any time.
+You can replace a proxy at any time, as it does not store any information about the clients that are connected to it.
+This process is handled using a reactivation key, which prevents you from losing the history of the proxy.
+If you do not use a reactivation key, the replacement proxy will become a new one with a new ID.
 The replacement proxy must have the same name and IP address as its predecessor.
 
+You can also re-register a proxy to change it from a traditional proxy to a Salt proxy.
```

Comment:
According to the later description, this does not seem possible.  We probably must test it.

Maybe, I'm getting the wording wrong.  I think we should avoid the word "re-register" etc., if we do not refer to make use of the a re-activation key.  Maybe, "register again" or "register newly" would be easier to understand.

Reply:
I would not use "re-register" because this could be misunderstood. I would say instead `you can also **reinstall** a proxy to change it .... . In this case, you won't need a reactivation key. See (link to Change a Proxy from Traditional to Salt).`